dough-items/src/main/java/io/github/bakedlibs/dough/items/nms/ItemNameAdapter19.java

```diff
@@ -0,0 +1,33 @@
+package io.github.bakedlibs.dough.items.nms;
+
+import io.github.bakedlibs.dough.reflection.ReflectionUtils;
+import io.github.bakedlibs.dough.versions.UnknownServerVersionException;
+import org.bukkit.inventory.ItemStack;
+
+import javax.annotation.ParametersAreNonnullByDefault;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
+public class ItemNameAdapter19 implements ItemNameAdapter {
```

Comment:
This should not be public.
```suggestion
class ItemNameAdapter19 implements ItemNameAdapter {
```